echopype/convert/set_groups_ek60.py

```diff
@@ -45,6 +47,9 @@ class SetGroupsEK60(SetGroupsBase):
         "gain_correction",
     }
 
+    # Variables that need beam and ping_time dimensions added to them.
+    beam_ping_time_names = set()
```

Comment:
This can be removed?

Reply:
For now, it needs to be there. The variable is passed to `set_groups_base.beam_groups_to_convention`, which expects a `set` type.